GX-common/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+# CMakeLists.txt for GreenX Common Modules, required by all other libs in
+# the package
+
+add_library(GXCommon "")
+
+set_target_properties(GXCommon
+        PROPERTIES
+        VERSION 0.0.1
+        SOVERSION 0.01
```

Comment:
- [ ] 0.0.1

Reply:
@jmbeuken Could you please not mark things as resolved unless the changes have been pushed, else it's easy to miss things. I have now made the change